js/app/widgets/slidingPanelOverlay.js

```diff
@@ -28,6 +28,7 @@ const SlidingPanelOverlay = new Knowledge.Class({
     },
 
     _init: function (props) {
+        this._transition_duration = 250;
```

Comment:
maybe worth moving the `250` into a constant since it is used twice in this file